iib/workers/tasks/build_merge_index_image.py

```diff
@@ -249,13 +251,22 @@ def handle_merge_request(
         )
 
         if deprecation_bundles:
-            deprecate_bundles(
-                deprecation_bundles,
-                temp_dir,
-                prebuild_info['binary_image'],
-                intermediate_image_name,
-                overwrite_target_index_token,
-            )
+            if is_fbc:
+                deprecate_bundles_fbc(
+                    bundles=deprecation_bundles,
+                    base_dir=temp_dir,
+                    binary_image=prebuild_info['binary_image'],
+                    from_index=intermediate_image_name,
+                    overwrite_target_index_token=overwrite_target_index_token,
```

Comment:
The function definition doesn't have `overwrite_target_index_token `. Should we remove it from here?

Reply:
removed